model/shared/src/main/scala/explore/model/ImagingConfigurationOptions.scala

```diff
@@ -24,16 +27,21 @@ sealed abstract class AvailableFilter {
   val range: Option[Quantity[Int, Nanometer]]
 }
 
+object AvailableFilter {
+  implicit val order: Order[AvailableFilter]       = Order.by(x => (x.centralWavelength, x.tag))
+  implicit val ordering: Ordering[AvailableFilter] = order.toOrdering
```

Comment:
I'm almost sure that cats provides an implicit conversion from `Order` to `Ordering` if you import `cats.syntax.all._` (probably?) at the use site.

Reply:
That's correct but you need to import `cats.implicits._` for that to work, Strange